worker/Dockerfile.debian

```diff
@@ -4,13 +4,13 @@ RUN apt-get update && apt-get install -y libssl-dev libsodium-dev
 COPY . .
 RUN cd worker; cargo build --release
 
-FROM debian:buster
+FROM debian:bullseye
 RUN echo 'deb http://deb.debian.org/debian buster-backports main' > /etc/apt/sources.list.d/backports.list
```

Comment:
This line can now be dropped, also replace `devscripts/buster-backports` with just `devscripts`.